packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -0,0 +1,358 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'localizations.dart';
+import 'picker.dart';
+
+/// Default aesthetic values obtained by comparing with iOS pickers.
+const double _kItemExtent = 32.0;
+const double _kPickerWidth = 330.0;
+/// Considers setting the default background color from the theme, in the future.
+const Color _kBackgroundColor = CupertinoColors.white;
```

Comment:
Is this right? We should let the user pick the background color.

Reply:
`CupertinoPicker` has colors fixed in `gradientScreen` (based on white background), so changing background color would make the picker looks weird.
https://github.com/doxuanviet1996/flutter/blob/countdown_timer/packages/flutter/lib/src/cupertino/picker.dart#L184

Having a different background color and we also have to change this gradient screen. Personally I think supporting black and white background is enough and pretty easy to add (reverting the color of `gradientScreen` would do it).

Reply:
Ah right. Ideally we should do https://github.com/flutter/flutter/issues/16831 but outside the scope of this PR.